app/models/manageiq/providers/redhat/infra_manager/provision/cloning.rb

```diff
@@ -47,6 +47,8 @@ def vm_clone_options
   end
 
   def disk_format
+    format_from_dialog =  get_option(:disk_format)
```

Comment:
Really minor but there is an extra space after the `= `

Reply:
done